### PR TITLE
fix(deploy-tooling): handle toggling of strict-ssl

### PR DESCRIPTION
**Pull request by longieirl**
Fix for https://github.com/SAP/open-ux-tools/issues/1121

- updated test cases

Negating options
https://github.com/tj/commander.js/blob/master/examples/options-negatable.js

### CODE REVIEW

File: packages/deploy-tooling/src/cli/index.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ export function createCommand(name: 'deploy' | 'undeploy'): Command {
         .option('--transport <transport-request>', 'Transport number to record the change in the ABAP system')
         .option('--name <bsp-name>', 'Project name of the app')
         .option('--strict-ssl', 'Perform certificate validation (use --no-strict-ssl to deactivate it)')
+        .option('--no-strict-ssl', 'Deactivate certificate validation')
```
Comment (Klaus-Keller) [inline packages/deploy-tooling/src/cli/index.ts:49]: Do we really need both, `--strict-ssl` and `--no-strict-ssl`? 
Should not one be the default and the other overwrites it? For security relevant things like this I'd say, strict ssl certificate validation is the default and by using option `--no-strict-ssl` you can deactivate it.

**Review by Klaus-Keller — COMMENTED** [review]
Added a question.
Additionally, I guess we also need to update the `README.md`.

**Review by devinea — APPROVED** [review]
Changes, changeset, tests and doc updates look good.

**Review by Klaus-Keller — APPROVED** [review]
- changes and test looks good
- did a manual test of `--no-strict-ssl`, no issue found

Approved from my side.

**Pull request merged**